src/accelerate/utils.py

```diff
@@ -621,6 +622,35 @@ def _pad_across_processes(tensor, dim=0, pad_index=0, pad_first=False):
     )
 
 
+def reduce(tensor: torch.Tensor, reduction="mean"):
+    """
+    Reduce `tensor` across all processes and perform a `reduction`.
```

Comment:
```suggestion
    Reduce a tensor across all processes by the mean of a given operation.
```
The arguments are described just below, so I like to keep the general description pure English :-)